web/src/App.js

```diff
@@ -22,7 +22,7 @@ const firebaseConfig = {
   appId: process.env.REACT_APP_FIREBASE_APP_ID,
   measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENT_ID
 };
-
+// CR * sharing with context/Redux
```

Comment:
When we start to integrate Firebase with our components a bit more, it may be useful to share the instance of Firebase throughout the component tree using Redux or a context component (probably Redux 😅).